packages/plugins/apollo-usage-report/src/index.ts

```diff
@@ -55,24 +58,68 @@ type ApolloUsageReportOptions = ApolloInlineTracePluginOptions & {
    * Client version to report to the usage reporting API
    */
   clientVersion?: StringFromRequestFn;
+  /**
+   * The version of the runtime (like 'node v23.7.0')
+   * @default empty string.
+   */
+  runtimeVersion?: string;
+  /**
+   * The hostname of the machine running this server
+   * @default $HOSTNAME environment variable
+   */
+  hostname?: string;
+  /**
+   * The OS identification string.
+   * The format is `${os.platform()}, ${os.type()}, ${os.release()}, ${os.arch()})`
```

Comment:
`navigator.platform` etc can be used in platform independent way, maybe?

Reply:
Oh I didn't know node had `navigator` API.
The thing I don't know is if Apollo is parsing this field or not. If it's paring the string, then we can't use `navigator.platform`.